ecosystem/sep-0006.md

```diff
@@ -387,6 +396,8 @@ For each withdrawal asset, it contains:
 * The fee structure
 * `min_amount`: Optional minimum amount. No limit if not specified.
 * `max_amount`: Optional maximum amount. No limit if not specified.
+* `fee_fixed`: Optional fixed (flat) fee for withdraw. In units of the deposited asset. Leave blank if there is no fee or the fee schedule is complex.
+* `fee_percent`: Optional percentage fee for withdraw. In percentage points. Leave blank if there is no fee or the fee schedule is complex.
```

Comment:
I think it would be better to have blank only indicate a complex fee schedule, with `0` needed here to indicate no fee. Wallets wouldn't want to request `/fee` unnecessarily (esp if it's not implemented), allowing blank to represent no fee would force a more complex check for whether to request `/fee` or not.

Reply:
@vcarl that's a great point. My thought was that wallets would use this stanza to decide if a complex fee structure is in play:

```
"fee": {
    "enabled": true
}
```

I think we need to keep the missing field implies zero semantics for backwards compatibility.